README.md

```diff
@@ -33,51 +70,39 @@ yarn deploy --network $NETWORK --gasprice $GAS_PRICE_WEI
 New files containing details of this deployment will be created in the `deployment` folder.
 These files should be committed to this repository.
 
-## Verify Contracts on Etherscan
+### Verify Contracts on Etherscan
 
-```
+```sh
 ETHERSCAN_API_KEY=<Your Key>
 yarn verify --network $NETWORK
 ```
 
-## Deployed Contract Addresses
+### Deployed Contract Addresses
 
-This package additonally contains a `networks.json` file at the root with the
-address of each deployed contract as well the hash of the Ethereum transaction
-used to create the contract.
+This package additonally contains a `networks.json` file at the root with the address of each deployed contract as well the hash of the Ethereum transaction used to create the contract.
 
-## Gas Reporter
-
-Gas consumption can be reported using by setting the `REPORT_GAS` flag when running tests as
-
-```sh
-REPORT_GAS=1 yarn test
-```
-
-## Solver Authentication
+### Solver Authentication
 
 This repo contains scripts to manage the list of authenticated solvers in all networks the contract has been deployed.
 
 The scripts are called with:
 
-```
+```sh
 yarn solvers command [arg ...]
 ```
 
 Here is a list of available commands.
-The commands flagged with [*] require the private key of the authentication contract owner to be available to the script, for example by exporting it with
-`export PK=<private key>`.
+The commands flagged with [*] require the private key of the authentication contract owner to be available to the script, for example by exporting it with `export PK=<private key>`.
 
 1. `add $ADDRESS` [*]. Adds the address to the list of registered solvers.
 2. `remove $ADDRESS` [*]. Removes the address from the list of registered
    solvers.
 3. `check $ADDRESS`. Checks if the given address is in the list of registered
    solvers.
 
-For example, adding the address `0x0000000000000000000000000000000000000042` to
-the solver list:
+For example, adding the address `0x0000000000000000000000000000000000000042` to the solver list:
```

Comment:
Might add here, "note that you must registered to add/remove solvers". Do we also have a yarn command to remove solvers?

Reply:
> Might add here, "note that you must registered to add/remove solvers".



> Do we also have a yarn command to remove solvers?

Yes:

> 2. `remove $ADDRESS` [*]. Removes the address from the list of registered solvers.

We already have a note on this above, let me know if you think this is enough or if we need another note.

> The commands flagged with [*] require the private key of the authentication contract owner to be available to the script, for example by exporting it with `export PK=<private key>`.